Gordon360/Controllers/ProfilesController.cs

```diff
@@ -438,6 +439,22 @@ public async Task<ActionResult<StudentProfileViewModel>> UpdateMobilePhoneNumber
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update office location (building description and room number)
+        /// </summary>
+        /// <param name="value">office location</param>
```

Comment:
Please explain clearly how office location is expressed.  I can see from the code below that there are two values separated by a comma, but would have to look farther in the code to see what the values are, and which is which.  The user of your API shouldn't have to hunt this info down, or guess.

It would probably be better to have the value be a JSON object, so the parsing can be automatic at both ends.

Reply:
Following Amos request I have changed the parameter of the UpdateOfficeLocation so that it is no longer a sttring with two values, but a JSON object for a new viewmodel I created